tracer/src/Datadog.Trace/LifetimeManager.cs

```diff
@@ -129,6 +129,8 @@ private void RunShutdownTasks()
                 {
                     SetSynchronizationContext(current);
                 }
+
+                DatadogLogging.CloseAndFlush();
```

Comment:
Wondering out loud: should this be _before_ we set the sync context back?

Reply:
It's not supposed to do anything async so it shouldn't matter